src/functions.js

```diff
@@ -238,13 +291,19 @@ var _ = Mavo.Functions = {
 	},
 
 	round: function(num, decimals) {
-		if (not(num) || not(decimals) || !isFinite(num)) {
-			return Math.round(num);
-		}
-
-		return +num.toLocaleString("en-US", {
-			useGrouping: false,
-			maximumFractionDigits: decimals
+		return Mavo.Script.binaryOperation(num, decimals, {
+			scalar: (num, decimals) => {
+				if (not(num) || not(decimals) || !isFinite(num)) {
+					return Math.round(num);
+				}
+		
+				return +num.toLocaleString("en-US", {
+					useGrouping: false,
+					maximumFractionDigits: decimals
+				});
+			},
+			leftUnary: b => b,
+			identity: 0
 		});
```

Comment:
Yeah, we can't really decide which argument should be an array based on the number of arguments... For example here, it doesn't really make sense for `decimals` to be an array. On the other hand, it doesn't hurt either, so I guess that's fine.